src/services/controllerservice/ControllerService.ts

```diff
@@ -397,7 +417,7 @@ export class ControllerService {
             });
         });
 
-        await this.serialPort.hardReset();
+        this.serialPort.hardReset();
```

Comment:
Why do we not wait for the setSignals to be completed?

Reply:
I originally had an await there but it seemed unnecessary, since the final gating factor is the receipt of data that is triggered by the signaling.  I think it works correctly either way.  If you think an await is better there, I will add it back.

Reply:
Okay, it's back, and the other one is gone.